config.js

```diff
@@ -24,5 +26,20 @@ module.exports = {
 		templates: "client/hbs",
 		upload: "upload"
 	},
+	categories:[
```

Comment:
I honestly don't know if I like that. I feel like there should be a fixed an agreed upon set of categories for consistencies sake